config.yaml

```diff
@@ -88,6 +88,10 @@ menu:
       url: /docs
       weight: 1
       parent: docs
+    - name: Languages
+      url: /docs/languages
+      weight: 2
+      parent: docs
     - name: Quick Start
```

Comment:
Why are quick start and tutorials still here?

Reply:
backwards compatibility?

Reply:
I'm trying to etch out meaningful updates to the site. This PR is about _adding_ a Languages section, and relocating individual QS and Tutorial pages.

Note that links to the section-root pages, [Quick start](https://grpc.io/docs/quickstart) and [Tutorial](https://grpc.io/docs/tutorials), exist in this site and elsewhere (e.g., grpc repo README pages). I didn't want to address their removal in the scope of this PR.

Of course, keeping these two section-root pages, doesn't mean that they have to appear in the Docs menu, but it seemed more in the spirit of this PR to leave them in for now.

Reply:
I 100% agree with leaving the old _pages_. But to have the navigation still in place to them is strange. When you click them they take you to a sort of no-man's land. Since they were removed from the side-nav, why would they remain on the top-nav? Unless we are leaving them there for a while to let users already familiar with the site find the new location of pages.

The future plan matters to me to approve the PR. If the plan is to "leave them as is, until we think of something else," then I would NAK this PR. If it is "oh, I forgot to, let me do that now" or "I plan to wait a week and then remove them," then that sounds fine.

Reply:
> ... leaving them there for a while to let users already familiar with the site find the new location of pages.

Yes.

> plan to wait ... and then remove them

Yes.

Reply:
@ejona86 - btw, while the "Quick start" and "Tutorial" entries still appear in the Docs drop-down menu, they do _not_ appear in the left-side nav.